tests/integration/targets/activation/tasks/main.yml

```diff
@@ -225,7 +225,7 @@
         name: "{{ event_stream_name }}"
         credential_name: "{{ credential_name_basic }}"
         organization_name: Default
-        event_stream_type: Basic Event Stream
+        event_stream_type: "basic"
```

Comment:
I'm not sure what are you trying to fix, but the right type is `Basic Event Stream`

Reply:
Ok, sorry, I mislead it with the credential type